### PR TITLE
[Feature] add copy credentials

### PR DESCRIPTION
**Pull request by Lily2point0**
- Add copy credentials to clipboard
- Add confirmation page

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -3,6 +3,8 @@ const express = require('express');
 const s3o = require('s3o-middleware');
 const helmet = require('helmet');
 const express_enforces_ssl = require('express-enforces-ssl');
+const path = require('path');
+const { copy } = require('copy-paste');
```
Comment (seanmtracey) [inline index.js:7]: Why the `{  }`? I think `const copy = require('copy-paste').copy` gives a little more insight into how the module works/is being used by the app.

File: project/ftlabs-ftda.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+				margin-right: 10px;
+			}
+        </style>
+    </head>
+    <body>
+    	<h1>FT Digital Archive Authenticator</h1>
+    	<p><i class="o-icons-icon o-icons-icon--tick"></i>Success</p>
+    	<p>You have been authenticated. Go back to the application and click 'Continue' to activate your credentials.</p>
+    </body>
+    <script async type="javascript" src="https://www.ft.com/__origami/service/build/v2/bundles/js?modules=o-fonts,o-icons"></script>
```
Comment (seanmtracey) [inline project/ftlabs-ftda.html:27]: Does this <script> need to be `async` if it's the last thing loaded in the document. Do we get a benefit from it? (asking for a friend)

File: index.js [hunk 3]
```diff
@@ -13,21 +15,32 @@ if(process.env.NODE_ENV !== 'local') {
...
+	}
+
+	if(response.key !== undefined) {
+		return copy(JSON.stringify(response), () => {
+			return res.sendFile(path.join(__dirname + '/project/' + req.params.project + '.html'));
+		});
 	}
 
-	return res.json({'error': 'You are not allowed to get keys for this resource.'});
+	res.status(response.errorType).send(response.error);
```
Comment (seanmtracey) [inline index.js:43]: You may need/want to set the `Content-Type` headers if `res.json()` is no longer being used.

**Review by seanmtracey — APPROVED** [review]

File: index.js (hunk 3 above)
Comment (Lily2point0) [reply index.js:43]: Sending a string back, not JSON.

File: project/ftlabs-ftda.html (hunk 2 above)
Comment (Lily2point0) [reply project/ftlabs-ftda.html:27]: From Origami documentation.

**Pull request merged**